audio/espeakedit/files/patch-espeakedit-mac.diff

```diff
@@ -0,0 +1,96 @@
+diff -ruEN orig/Makefile src/Makefile
+--- orig/Makefile	2020-06-25 21:54:53.000000000 +0700
++++ src/Makefile	2020-06-26 09:34:19.000000000 +0700
+@@ -10,17 +10,18 @@
+ 
+ OBJS=$(patsubst %.cpp,%.o,$(SRCS))
+ 
+-WX_LIBS = -pthread   `wx-config --libs`
++WX_LIBS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --libs`
++
++WX_CFLAGS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --cflags`
```

Comment:
You can't hardcode /opt/local/Library/Frameworks; you have to arrange (using a placeholder in the patch and then a reinplace in the portfile) for the value of ${frameworks_dir} to be used.